src/main/java/ch/njol/skript/entity/EntityData.java

```diff
@@ -503,6 +522,7 @@ public E spawn(Location location, org.bukkit.util.@Nullable Consumer<E> consumer
 	@Nullable
 	public E spawn(Location location, @Nullable Consumer<E> consumer) {
 		assert location != null;
+		if (!canSpawn(location.getWorld())) return null;
```

Comment:
It's likely that we need to make sure the location's world is not null

Reply:
null check added :)